src/scripts/macros/treat-wounds.ts

```diff
@@ -150,13 +151,16 @@ async function applyChanges(actor: CreaturePF2e, $html: JQuery, event: JQuery.Tr
                 ChatMessagePF2e.create({
                     type: CONST.CHAT_MESSAGE_TYPES.ROLL,
                     flavor: `<strong>${game.i18n.localize("PF2E.Actions.TreatWounds.Rolls.RiskySurgery")}</strong>`,
-                    rolls: [(await new Roll("{1d8}[slashing]").roll({ async: true })).toJSON()],
+                    rolls: [(await new DamageRoll("{1d8[slashing]}").roll({ async: true })).toJSON()],
                     speaker,
                 });
             }
 
             if (healFormula) {
-                const healRoll = await new Roll(`{${healFormula}}[healing]`).roll({ async: true });
+                const fullFormula = healFormula.includes("+")
```

Comment:
Can't this use the first formula no matter what? What's wrong with a DamageRoll that's `{(1d6)[healing]}`?

Reply:
Doesn't it get prettified anyways?